src/frontend/js/hooks/useJoanieUserProfile.tsx

```diff
@@ -0,0 +1,30 @@
+import { defineMessages } from 'react-intl';
+import { API } from 'types/Joanie';
+import { useJoanieApi } from 'contexts/JoanieApiContext';
+import { JoanieUserProfile } from 'types/User';
+import { ResourcesQuery, useResource, UseResourcesProps } from './useResources';
+
+const messages = defineMessages({
+  errorGet: {
+    id: 'hooks.useJoanieUserProfile.errorGet',
+    description: 'Error message shown to the user when user fetch request fails.',
+    defaultMessage:
+      'An error occurred while fetching user profile information. Please retry later.',
+  },
+  errorNotFound: {
+    id: 'hooks.useJoanieUserProfile.errorNotFound',
+    description: "Error message shown to the user when it isn't logged.",
+    defaultMessage: "You aren't logged in.",
+  },
+});
+
+const props: UseResourcesProps<JoanieUserProfile, ResourcesQuery, API['user']['me']> = {
+  queryKey: ['profile'],
+  apiInterface: () => useJoanieApi().user.me,
+  session: true,
+  messages,
+};
+/**
+ * Joanie Api hook to retrieve the authenticated user.
+ */
+export const useJoanieUserProfile = () => useResource(props)(undefined, {}, { enabled: true });
```

Comment:
We could enable this query only when user is authenticated.

```suggestion
export const useJoanieUserProfile = () => useResource(props)(undefined, {}, { enabled: undefined });
```

Or in a way more explicit : 

```suggestion
export const useJoanieUserProfile = () => {
	const { user } = useSession()
	return useResource(props)(undefined, {}, { enabled: !!user });
}
```

Reply:
i first did that and it required to code useJoanieUserAbilities differently.
I choose this way to do it but i'll change if your don't like it.